openscm/openscm.py

```diff
@@ -0,0 +1,11 @@
+from .core import Core
+
+
+class OpenSCM(Core):
+    """
+    OpenSCM class.
+
+    Represents model runs with a particular simple climate model.
```

Comment:
What do we actually want this class to do? Be the container for a single run? So it would have attribtues like `_adapter` (a, potentially already intialised, adapter instance), `_input` (a `ParameterSet` which gets passed to `_adapter._initialise_model_inputs` etc.) and `_output` (which reads from a `ParameterSet` which the adapter creates for output?). Then for ensembles, you just stitch lots of these together?

Reply:
that's the more pythonic convenience layer (though i would prefer to see the core interface as the standard way to use openscm). implementation will be a dedicated pull request.

Reply:
Ok but it's the same as `Core`, just more pythonic? 

If yes, do we want to tweak the diagram in the docs so that it's clear that all that `OpenSCM` does is basically wrap `Core` in a more Pythonic way?

Reply:
Sorry I should probably look at the diagram properly before commenting. All good!